src/js/tabs/configuration.js

```diff
@@ -1215,19 +782,23 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             }
 
             FC.PID_ADVANCED_CONFIG.pid_process_denom = value;
-
-            FC.PID_ADVANCED_CONFIG.digitalIdlePercent = parseFloat($('input[name="digitalIdlePercent"]').val());
             if (semver.gte(FC.CONFIG.apiVersion, "1.25.0") && semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_41)) {
                 FC.PID_ADVANCED_CONFIG.gyroUse32kHz = $('input[id="gyroUse32kHz"]').is(':checked') ? 1 : 0;
             }
 
-            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_31)) {
-                FC.RX_CONFIG.fpvCamAngleDegrees = parseInt($('input[name="fpvCamAngleDegrees"]').val());
-            }
+            FC.RX_CONFIG.fpvCamAngleDegrees = parseInt($('input[name="fpvCamAngleDegrees"]').val());
 
             analytics.sendChangeEvents(analytics.EVENT_CATEGORIES.FLIGHT_CONTROLLER, self.analyticsChanges);
             self.analyticsChanges = {};
 
+            // fill some data
+            FC.GPS_CONFIG.auto_baud = $('input[name="gps_auto_baud"]').is(':checked') ? 1 : 0;
```

Comment:
Where have the version checks for this gone?

Reply:
Added